shunit2_test_helpers

```diff
@@ -226,6 +226,13 @@ _th_showOutput() {
   unset _th_return_ _th_stdout_ _th_stderr_
 }
 
+# Constants used to test file and directory assertions
+TH_EXISTING_DIRECTORY=$(exec 2>/dev/null;cd -- "$(dirname "$0")" || exit; unset PWD; /usr/bin/pwd || /bin/pwd || pwd)
+TH_EXISTING_FILE="${TH_EXISTING_DIRECTORY}/$0"
```

Comment:
Same here. Use an explicit filename (name of your choice) inside of ${SHUNIT_TMP}. $0 is dynamic, and one cannot search for the resulting filename in the code and find it, making it more difficult to debug the code.

In general with unit tests, be as explicit as possible.